devel/simple-web-server/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.0
+
+name	  	    simple-web-server
+version		    3.0.0-rc3
+categories          devel
+license             MIT
+maintainers         gmail.com:rene.vanpaassen
+platforms	    darwin
+description         HTTP and HTTPS server and client library in C++11
+long_description    A very simple, fast, multithreaded, platform independent \
+		    HTTP and HTTPS server and client library implemented \
+		    using C++11 and Boost.Asio. Created to be an easy way \
+		    to make REST resources available from C++ applications.
+
+homepage            https://gitlab.com/eidheim/Simple-Web-Server
+master_sites	    https://gitlab.com/eidheim/Simple-Web-Server/-/archive/v${version}/
+distname	    Simple-Web-Server-v${version}
+
+checksums           rmd160 d757957fd491b0f7fe8ea6cb9cfba4a133174b2a \
+                    sha256 b6230fb1f267cf7f45b6b8eb6efb3ed6a4308fd48003dc8397eba1d7cf048467 \
+		    size    30857
+
+depends_build	    port:boost \
+		    port:cmake
```

Comment:
Use `depends_build-append` here and remove `port:cmake`. The cmake portgroup adds the correct dependency but you were overriding that.

Reply:
thanks. Did that